cmake/common.cmake

```diff
@@ -602,6 +602,8 @@ function(add_perf_test)
   set(TESTS_SUFFIX "")
   if("sgx" IN_LIST COMPILE_TARGETS)
     set(TESTS_SUFFIX "${TESTS_SUFFIX}_sgx")
+  else()
+    set(TESTS_SUFFIX "${TESTS_SUFFIX}_virtual")
```

Comment:
This looks like a coarse distinction. Can we instead have both `ls_sgx_cft` and `ls_virtual_cft` if we built both (choosing between them by adding a `--enclave-type` arg to the python script)?

Reply:
done